Code/GraphMol/Descriptors/testATF.cpp

```diff
@@ -0,0 +1,59 @@
+//  Created by Guillaume GODIN
+//  Copyright (C) 2012-2017 Greg Landrum
+//   @@ All Rights Reserved @@
+//
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDGeneral/test.h>
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/MolWriters.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <RDGeneral/RDLog.h>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+#include <sstream>
+
+#include <chrono>  // for high_resolution_clock
+
+#include <GraphMol/Descriptors/AtomFeat.h>
+
+void test1() {
+ 
+  
+    RDKit::ROMOL_SPTR m( RDKit::SmilesToMol( "CO") );    
```

Comment:
it's quicker and nicer to create molecules for testing like this:
```suggestion
    auto m = "CO"_smiles;    
```
That returns an `std::unique<ptr>`, so you don't need to worry about memory.